templates/object-cache.php

```
@@ -212,7 +212,14 @@ public function add( $key, $data, $group = 'default', $expire = 0 ) {
 			$local_key = $this->get_key( $key, $group );
 
 			if ( $expire ) {
-				return $this->redis->set( $local_key, $data, array( 'nx' => true, 'ex' => $expire ) );
+				return $this->redis->set(
```

PHPStan (2.2.5)

[warning] Cannot call method set() on Redis|RedisCluster|null.


(method.nonObject)